spec/features/team/invite_a_team_member_spec.rb

```diff
@@ -49,7 +49,7 @@
         end
       end
 
-      context "with non gov.uk email address" do
+      context "for new user with non gov.uk email address" do
```

Comment:
Anything not related to the actual issue should be in a separate PR. This just makes the PR unnecessarily more complicated.

Reply:
@Renny4Real these still need to be removed from this PR. Changes to line 54 and line 40.